cellengine/resources/gate.py

```diff
@@ -89,10 +90,25 @@ def update_family(experiment_id, gid: str, body: Dict):
         if res["nModified"] < 1:
             raise Warning("No gates updated.")
 
-    def tailor_to(self, fcs_file: FcsFile):
-        self.tailored_per_file = True
-        self.fcs_file_id = fcs_file._id
-        self.update()
+    def apply_tailoring(self, fcs_file_ids: List[str], **kwargs):
+        """Tailor this gate to a specific FCS file or files."""
```

Comment:
I don't think `**kwargs` are applied to a copy of the self gate, right?

Reply:
Yup, you're right, nice catch. I actually don't think there's any additional params that can be passed to [the endpoint](https://docs.cellengine.com/api/#apply-tailoring), so I've removed `**kwargs`